ADRs/RH/SIG-SRE/ADR-00006 SLO RACI Chart.md

```diff
@@ -0,0 +1,89 @@
+# ADR-00006 SLO RACI Chart
+
+Authors: Lisa Seelye <@lisa>, Jeremy Eder
+
+
+## Status
+
+Draft
+
+
+## Problem Statement
+
+With [a large number of personas][personas-adr] involved in a managed services context it can be difficult to keep track of how each persona interacts with or is involved with the many SLO processes.
+
+## Goals
+
+Create a matrix that breaks down the steps involved with the SLO lifecycle and use the [RACI model][wiki-raci] to create a reference matrix for personas.
+
+## Non-goals
+
+Inclusion of every persona and every possible step is not desired. Some personas are not involved with this matrix and it is not feasible to enumerate every possible step.
+
+
+## Current Architecture
+
+Ad-Hoc.
+
+## Proposed Architecture
+
+We propose to introduce a RACI, or Responsibility Assignment Matrix to define and help visualize roles typically assigned to [various personas][personas-adr] involved with the SLO lifecycle. “RACI” refers to the four different roles, discussed below. These roles are then associated with one or more steps in a table to form the matrix, or chart.
+
+### Proposed Roles
+
+The four roles are Responsible (R), Accountable (A), Consulted (C) and Informed (I).
+
+#### Responsible
+
+The Responsible role applies to those who are expected to do the work.
+
+#### Accountable
+
+The Accountable role applies to those who are answerable for the work being done.
+
+#### Consulted
+
+The Consulted (or Consultant) role applies to those whose opinions are sought. They will likely be subject matter experts, or those whose input is especially important.
+
+#### Informed
+
+The Informed role applies to those who are only notified of the status. Their input is not required or even necessarily sought out; this can be a one-way communication.
+
+### Responsibility Matrix
+
+The following matrix can be used as a starting point and is intended to be copied to other documents with names filled in for those personas. The steps in the matrix correspond to activities taking place in the phases from the [SLO Lifecycle][slo-lifecycle-adr] and areas of responsibility to each persona.
+
+**Note**: Not every [persona][personas-adr] may appear in this chart.
+
+| Step                 | Service Owner | Product Owner(s) | Engineering/Quality Lead | Sw. Eng/QE | SRE IC | Eng Manager/Director | Exec (VP) |
```

Comment:
Outstanding question: How do we cater for orgs where they have very different "scenarios" e.g. one service or many services; different engineering structures and size; different levels of maturity.

Reply:
There's two things going on here:

1) The ADR as written suggests a 1-to-1 mapping of services to teams. This may not map to the "outside world" where one team may be responsible for multiple services, or a single service may have multiple teams responsible for it.

In the first situation I think the ADR still applies because there could still be people with the personas to fill the required roles (see the second "thing").

In the second sitaution I think we need to be a little more clever about how we reckon with things. I think we'd like to identify the people who fit these peronas and loop them into a "virtual team." If we can do that, then the ADR still follows on logically.

To me, I think the ADR still applies even if there aren't fixed teams because at some basic level I feel there are still these personas that exist and drive "action." It's that circle of folks that need to be involved here.

2) I think the real concern for this second "thing" is how to handle orgs which do not have "as wide" a structure with all of these personas. They might not be able to easily reckon with how we have set up the personas. It may not be clear in the persona doc or here how to reckon with one person inhabiting multiple personas.

I don't know where best to add the clarification that one person may inhabit multiple personas, or that they may simply not exist in the org structure. There's an argument to be made for the personas doc and this doc, to a lesser extent perhaps.

Reply:
@lisa I don't think it is detrimental to emphasize this point ("one person may inhabit multiple personas, or that they may simply not exist in the org structure") in both docs as long as we are in sync.

IMO, this is not something that should necessarily hold us up... as long as we stress the above point.

Reply:
IMO, I think it is enough. IMO we don't want it to become too "complicated" as it becomes unwieldy and folks tend not to read it.

Reply:
@brizrobbo Sounds good. I'll resolve this conversation for the time being.